renovate.json

```diff
@@ -9,22 +9,22 @@
   ],
   "packageRules": [
     {
-      "matchPackageNames": ["*"],
+      "matchPackagePatterns": ["*"],
```

Comment:
do we need it if it's wildcard? 
can't tell from docs if it's required or not 
`Use this field if you want to have one or more package names patterns in your package rule.` makes it seem optional

Reply:
Tested it on a fork and I got this: 
<img width="961" alt="image" src="https://github.com/WalletConnect/web3modal/assets/1979423/2746fe9b-28f3-44c8-a2e7-983a51b546b1">